.travis.yml

```diff
@@ -2,22 +2,20 @@ language: ruby
 sudo: false
 
 rvm:
-  - 2.2.5
-  - 2.3.1
+  - 2.4.9
+  - 2.5.7
+  - 2.6.5
 
 gemfile:
-  - Gemfile.rails42
   - Gemfile.rails50
   - Gemfile.rails51
```

Comment:
Looks like these aren't supported either.  We could test 6.0 and rails master instead.

Reply:
Right, I think 5.2, 6.0, and head is good enough. 5.2 is still partially supported, see https://guides.rubyonrails.org/maintenance_policy.html.

Reply:
Yeah, I didn't mean to imply 5.2 wasn't supported.  This comment was only on the Gemfile.rails50 and Gemfile.rails51 lines